### PR TITLE
Archive incomplete genotype results

### PR DESCRIPTION
**Pull request by mgcam**
Built on top of https://github.com/wtsi-npg/npg_qc/pull/841

### CODE REVIEW

File: t/60-autoqc-db_loader.t [hunk 1]
```diff
@@ -874,4 +874,18 @@ subtest 'loading review and other results from path' => sub {
...
+subtest 'loading partially defined results' => sub {
+  plan tests => 2;
+ 
+  my $db = $db_helper->create_test_db(q[npg_qc::Schema], 't/data/fixtures');
+  my $db_loader = npg_qc::autoqc::db_loader->new(
+    path    => ['t/data/autoqc/dbix_loader/short_results'],
+    schema  => $db,
+    verbose => 0,
+  );
+  lives_ok { $db_loader->load() } 'no error loading an ncomplete result';
```
Comment (nerdstrike) [inline t/60-autoqc-db_loader.t:886]: s/ncomplete/incomplete/

File: lib/npg_qc/autoqc/checks/genotype.pm [hunk 2]
```diff
@@ -658,7 +661,7 @@ sub _set_attrib_by_geno_refset {
   my $name   = fileparse($self->geno_refset);
 
   $self->_set_sequenom_plex($name);
-  $self->_set_snp_call_set($name);
+  $self->_set_ssnp_call_setnp_call_set($name);
```
Comment (nerdstrike) [inline lib/npg_qc/autoqc/checks/genotype.pm:664]: This invocation doesn't look right, and doesn't turn anything up in devel code.

**Review by nerdstrike — CHANGES_REQUESTED** [review]
Confused by one line, otherwise looks good.

File: lib/npg_qc/autoqc/checks/genotype.pm (hunk 2 above)
Comment (mgcam) [reply lib/npg_qc/autoqc/checks/genotype.pm]: This is an accidental edit, the code is now reverted back to what it was. The fact that the tests still pass point to low test coverage. Thank you very much for spotting this error.

**Review by nerdstrike — APPROVED** [review]

**Pull request merged**